tests/python_on_whales/components/test_compose.py

```
@@ -894,6 +894,37 @@ def test_compose_logs_follow():
     docker.compose.down(timeout=1)
 
 
+def test_events():
+    docker = DockerClient(
+        compose_files=[
+            PROJECT_ROOT
+            / "tests/python_on_whales/components/compose-with-healthcheck.yml"
+        ],
+        compose_compatibility=True,
+    )
+
+    docker.compose.up(["unhealthy_service"], detach=True)
+
+    events_iterator = docker.compose.events(["unhealthy_service"])
+
+    events = []
+    try:
+        for event in events_iterator:
+            events.append(event)
+            if len(events) >= 3:  # Collect a few events for testing
+                break
+    except KeyboardInterrupt:

```

I'm not sure why the try except is there. If there is a keyboard interrupt, why would we want to swallow the exception?

---

You're right, the try/except block is not necessary here. I'll remove it.